frontend/packages/modelServing/extension-points/index.ts

```diff
@@ -12,10 +12,12 @@ import type { ModelServingPodSpecOptionsState } from '@odh-dashboard/internal/co
 import type { K8sResourceCommon } from '@openshift/dynamic-plugin-sdk-utils';
 import type { InferenceServiceModelState } from '@odh-dashboard/internal/pages/modelServing/screens/types';
 import type { ComponentCodeRef } from '../../plugin-core/src/extension-points/types';
+import { ToggleState } from '@odh-dashboard/internal/components/StateActionToggle';
 
 export type DeploymentStatus = {
   state: InferenceServiceModelState;
   message?: string;
+  stoppedStates?: ToggleState;
```

Comment:
hate this name and not sure if it makes sense to live under here

Reply:
maybe we removed the layer for `stoppedStatus` and just union with the ToggleState so it becomes 
```suggestion
  export type DeploymentStatus = {
      state: InferenceServiceModelState;
      message?: string;
      isStarting: boolean;
      isStopping: boolean;
      isRunning: boolean;
      isStopped: boolean;
```